code/src/UserRating.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+const rating = [
+  {
+  value:"poor",
+  label:"Poor",
+  id:"poor"
+  },
+  {
+    value:"very poor",
+    label:"Very poor",
+    id:"very poor"
+  },
+  {
+    value:"average",
+    label:"Average",
+    id:"average"
+  },
+  {
+    value:"good",
+    label:"Good",
+    id:"good"
+  },
+  {
+    value:"excellent",
+    label:"Excellent",
+    id:"excellent"
+  }
+  // 'Average',
+  // 'Good',
```

Comment:
Commented code should be deleted as well

Reply:
Deleted commented codes